openapi.yaml

```
@@ -1992,6 +1994,7 @@ paths:
           response:
             body:
               batch_id: '1235'
+              run_id: "1234-1235"
```

```suggestion
              run_id: "a1b2c3d4-e5f6-7890-abcd-ef1234567890"
```

Recommend using this obviously-fake-but-with-real-syntax value, which is what I use in the PR that improves sample code. When I saw `1234-1235` I first thought it was a range of values and was a little confused.

I see we already had `1234-1235` in this file as a `run_id` value elsewhere in the definition of the Get Run Results operation, so I think it’s worth changing all occurrences in this operation to use the longer, more realistic value.